cla_frontend/apps/status/views.py

```diff
@@ -31,8 +23,17 @@ def smoketests_json(request):
     """
     Run smoke tests and return results as JSON datastructure
     """
-
     from cla_frontend.apps.status.tests.smoketests import SmokeTests
+    return JsonResponse(smoketest(SmokeTests))
 
-    return JSONResponse(smoketest(SmokeTests))
 
+class PingJsonView(View):
+    """
+    Stub IRaT PingJsonView for compatibility with current and imminent infra changes
```

Comment:
📝Notes for future: "imminent infra changes" means moving to Kubernetes, where we will be able to view these details from the environment.